src/pages/home.js

```diff
@@ -1,10 +1,9 @@
-import "./font.css";
 import NavBar from "../components/navBar";
 import { Link } from "react-router-dom";
 
 function Home() {
   return (
-    <div className="bg-cod-gray font absolute h-full w-full">
+    <div className="bg-cod-gray absolute h-full w-full font-['audioWide']">
```

Comment:
Not an issue, but doesn't `font-audioWide` work, without the brackets?

Reply:
Here it worked, but I changed it on the `question.js` and on the OpenSans I had to have the brackets to work

Reply:
I experimented around and if it's child it appears it needs to have brackets, if I had OpenSans on the place of audioWide it worked.